desktop/analytics/partner.js

```diff
@@ -22,4 +22,19 @@
       partner_slug: $(e.currentTarget).data('partner-slug')
     })
   })
+
+  $('.main-layout-container').on('click', '#partner2-contact .email-gallery', function (e) {
+    analytics.track('Click',{
+      partner_id: $(e.currentTarget).data('partner-id'),
+      label: "Contact gallery by email"
+    })
+  })
+
+  $('.main-layout-container').on('click', '#partner2-contact .partner2-website', function (e) {
```

Comment:
Excatly. I tried with `$('.partner2-content')` initially, but it works only when you load the contact page directly.